draft-pauly-add-resolver-discovery.md

```diff
@@ -411,6 +446,13 @@ Meaning:
 Reference:
 : This document.
 
+## Special Use Domain Name "resolver.arpa"
+
+This document calls for the creation of the "resolver.arpa" SUDN. This will allow resolvers to respond to 
+queries directed at themselves rather than a specific domain name. While this document uses "resolver.arpa"
+to return HTTPSSVC records indicating DoH capability, the name is generic enough to allow future reuse for
+other purposes where the resolver wishes to provide information about itself to the client. 
+
 # Acknowledgments
 
 Thanks to Erik Nygren, Lorenzo Colitti, Tommy Jensen, Mikael Abrahamsson,
```

Comment:
Let's add you to the author list, and we can remove the thanks =)

Reply:
Done! Thanks for letting me work with you all :)